### PR TITLE
Update console startup and help url for PowerShell docs

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

PS7 still shows https://aka.ms/pscore6-docs URL which can be confusing.  Updated to new generic https://aka.ms/powershell-docs URL.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx [hunk 1]
```diff
@@ -121,7 +121,7 @@
     <value>PowerShell {0}
 Copyright (c) Microsoft Corporation. All rights reserved.
 
-https://aka.ms/pscore6-docs
+https://aka.ms/powershell-docs
```
Comment (bergmeister) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: Why not https://aka.ms/powershell which is more high level and might be more useful to users whilst they can still get to the https://aka.ms/powershell-docs from there

Comment (rkeithhill) [reply src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: ^ @bergmeister  That does seem to be a more appropriate landing page.

Comment (SteveL-MSFT) [reply src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: @joeyaiello thoughts?

Comment (SteveL-MSFT) [reply src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: I'm ok changing this to the top level link.

**Review by adityapatwardhan — APPROVED** [review]

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx [hunk 2]
```diff
@@ -121,7 +121,7 @@
     <value>PowerShell {0}
```
Comment (PoshChan) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: @SteveL-MSFT, your last commit had 10 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

TabCompletion.Should complete abbreviated cmdlet
```powershell
Expected exactly 'Import-PowerShellDataFile', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 16
16:         $res | Should -BeExactly 'Import-PowerShellDataFile'
```
TabCompletion.Should complete abbreviated function
```powershell
Expected the actual value to be greater than or equal to 1, but got 0.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 21
21:         $res.Count | Should -BeGreaterOrEqual 1
```
Temp: drive.TEMP: drive exists
```powershell
Expected exactly 'Temp', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-PSDrive.Tests.ps1: line 64
64:         $res.Name | Should -BeExactly "Temp"
```
Get-Command Feature tests.-UseAbbreviationExpansion tests.Can return multiple results relying on auto module loading
```powershell
Expected a collection with size 2, but got an empty collection.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Command.Tests.ps1: line 39
39:             $results | Should -HaveCount 2
```
Get-Command Feature tests.-UseAbbreviationExpansion tests.Can return multiple results for cmdlets matching abbreviation
```powershell
Expected 'Invoke-Command' to be found in collection @(), but it was not found.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Command.Tests.ps1: line 69
69:             $results.Name | Should -Contain "Invoke-Command"
```

**Comment by SteveL-MSFT** [comment]
@PoshChan please retry macos,windows

**Comment by PoshChan** [comment]
@SteveL-MSFT, successfully started retry of `PowerShell-CI-macOS, PowerShell-CI-Windows`

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx (hunk 2 above)
Comment (PoshChan) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: @SteveL-MSFT, your last commit had 10 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

TabCompletion.Should complete abbreviated cmdlet
```powershell
Expected exactly 'Import-PowerShellDataFile', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 16
16:         $res | Should -BeExactly 'Import-PowerShellDataFile'
```
TabCompletion.Should complete abbreviated function
```powershell
Expected the actual value to be greater than or equal to 1, but got 0.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 21
21:         $res.Count | Should -BeGreaterOrEqual 1
```
Temp: drive.TEMP: drive exists
```powershell
Expected exactly 'Temp', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Get-PSDrive.Tests.ps1: line 64
64:         $res.Name | Should -BeExactly "Temp"
```
Get-Command Feature tests.-UseAbbreviationExpansion tests.Can return multiple results relying on auto module loading
```powershell
Expected a collection with size 2, but got an empty collection.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Command.Tests.ps1: line 39
39:             $results | Should -HaveCount 2
```
Get-Command Feature tests.-UseAbbreviationExpansion tests.Can return multiple results for cmdlets matching abbreviation
```powershell
Expected 'Invoke-Command' to be found in collection @(), but it was not found.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Command.Tests.ps1: line 69
69:             $results.Name | Should -Contain "Invoke-Command"
```

**Comment by adityapatwardhan** [comment]
Close and re-open to force Windows CI to pickup build failure workaround.

**Comment by adityapatwardhan** [comment]
@PoshChan please remind me in 1 hour

**Comment by PoshChan** [comment]
@adityapatwardhan, this is the reminder you requested 1 hour ago

**Pull request merged**